manifests/params.pp

```diff
@@ -1,7 +1,7 @@
 # Params class for parent foreman_scap_client
 # @api private
 class foreman_scap_client::params {
-  $downcased_fqdn = downcase($facts['fqdn'])
+  $downcased_fqdn = $trusted['certname']
```

Comment:
I think you can actually inline them on line 37 and 38 now.

Reply:
Mmm good point, simplified the file and put them inline.